samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -239,5 +239,48 @@ public class Assignment
                 throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
             }
         }
+
+        //Create a draft assignment 
+        public static async Task<EducationAssignment> PatchAsync(
+            GraphServiceClient client,
+            string classId,
+            string assignmentId)
+        {
+            try
+            {
+              
+                var requestBody = new EducationAssignment
+                {
+                    DisplayName = "Reading and review test updated",                    
+                };
+
+                // Get a Graph client using delegated permissions
```

Comment:
remove this comment

Reply:
removed the line 257 and a blank line space